scripts/deploy.ts

```diff
@@ -235,6 +239,16 @@ async function deploySessionKeyManagerModule(deployerInstance: Deployer) {
   );
 }
 
+async function deployBatchedSessionRouterModule(deployerInstance: Deployer) {
+  await deployGeneric(
+    deployerInstance,
+    DEPLOYMENT_SALTS.BATCHED_SESSION_ROUTER_MODULE,
```

Comment:
@ankurdubey521 we can review the salt for it for updated contract deployment.

Reply:
Noted